File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/platform/bus/Lm75BusConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.platform.bus;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.PlatformType;
import lombok.Data;

@Data
public class Lm75BusConfig extends I2cBusConfig {
    public Lm75BusConfig() {
        super();
        setPlatform(PlatformType.LM75);
    }
}

File: config/editor/src/app/definition/platform/bus/Lm75BusConfig.ts
import {I2cBusConfig} from "./I2cBusConfig";
import {PlatformType} from "../../enums/PlatformType";

export class Lm75BusConfig extends I2cBusConfig {
    public constructor(busId: string, address: number) {
        super(PlatformType.LM75,busId,address);
    }
}
